<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>695</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>695</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>695</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_setpoint">
      <property name="text">
       <string>Height Setpoint</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3" alignment="Qt::AlignHCenter">
     <widget class="QSlider" name="slider_current">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>130</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3" alignment="Qt::AlignHCenter">
     <widget class="QDoubleSpinBox" name="spin_current">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <double>2.000000000000000</double>
      </property>
      <property name="maximum">
       <double>15.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QSlider" name="slider_setpoint">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>26</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="button_BTstop">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>BT Stop</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="button_BTstart">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>BT Start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter">
     <widget class="QDoubleSpinBox" name="spin_setpoint">
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>2.000000000000000</double>
      </property>
      <property name="maximum">
       <double>15.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_current">
      <property name="text">
       <string>Current Height</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="3">
     <widget class="QWidget" name="widget" native="true">
      <widget class="QCustomPlot" name="plot_inFlow" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>604</width>
         <height>309</height>
        </rect>
       </property>
      </widget>
      <widget class="QCustomPlot" name="plot_outFlow" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>313</y>
         <width>604</width>
         <height>309</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" rowspan="4">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>slider_setpoint</tabstop>
  <tabstop>slider_current</tabstop>
  <tabstop>spin_setpoint</tabstop>
  <tabstop>spin_current</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
